appengine/endpoints-frameworks-v2/discovery/pom.xml

```diff
@@ -0,0 +1,124 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>war</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <groupId>com.example.helloendpoints</groupId>
+    <artifactId>helloendpoints</artifactId>
+
+    <properties>
+        <!-- Uncomment to use Endpoints Frameworks v1.0 -->
+        <!--
+        <appengine.endpoints.version>1.9.48</appengine.endpoints.version>
+        -->
+        <!-- End of Endpoints Frameworks v1.0 -->
+        <appengine.maven.version>1.0.0</appengine.maven.version>
+        <endpoints.framework.maven.version>1.0.0-beta2</endpoints.framework.maven.version>
+        <endpoints.framework.version>2.0.0-beta.12</endpoints.framework.version>
+        <javax.inject.version>1</javax.inject.version>
+        <jdo.api.version>3.0.1</jdo.api.version>
+        <mojo.versions.maven.version>2.1</mojo.versions.maven.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <servlet.api.version>2.5</servlet.api.version>
+    </properties>
+
+    <prerequisites>
+        <maven>3.3.9</maven>
+    </prerequisites>
+
+    <dependencies>
+        <!-- Compile/runtime dependencies -->
+        <!-- Uncomment to use Endpoints Frameworks v1.0 -->
+        <!--<dependency>
+              <groupId>com.google.appengine</groupId>
+              <artifactId>appengine-endpoints</artifactId>
+              <version>${appengine.endpoints.version}</version>
+        </dependency>-->
+        <!-- End of Endpoints Frameworks v1.0 -->
+        <!-- Endpoints Frameworks v2.0 -->
+        <dependency>
+            <groupId>com.google.endpoints</groupId>
+            <artifactId>endpoints-framework</artifactId>
+            <version>${endpoints.framework.version}</version>
+        </dependency>
+        <!-- End of Endpoints Frameworks v2.0 -->
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>servlet-api</artifactId>
+            <version>${servlet.api.version}</version>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>javax.inject</groupId>
+            <artifactId>javax.inject</artifactId>
+            <version>${javax.inject.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>javax.jdo</groupId>
```

Comment:
Good catch. The sample doesn't use SQL whatsoever. I missed this dependency during clean up.